spark-indexer/src/main/java/org/uniprot/store/spark/indexer/publication/mapper/MappedReferencesToPublicationDocumentBuilderConverter.java

```diff
@@ -35,6 +35,7 @@
         implements PairFunction<
                 Tuple2<String, Iterable<MappedReference>>, String, PublicationDocument.Builder> {
     private static final long serialVersionUID = -5482428304872200536L;
+    private static final String UNCLASSIFIED = "Unclassified";
 
```

Comment:
If we could move UNCLASSIFIED category to PublicationCategory enum then all the Publication's categories will be at one place.